tests/modules/fastani/main.nf

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { FASTANI } from '../../../modules/fastani/main.nf' addParams( options: [:] )
+
+workflow test_fastani {
+
+    query = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+    reference = file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
```

Comment:
For the time being, I've simply relied upon the available fasta files - please do let me know if you'd like to wait till we have bacterial genomes in `nf-core/test-datasets` instead?

Reply:
I think for FASTANI, this will work just fine.